Jenkinsfile

```diff
@@ -96,7 +96,7 @@ void build_taichi() {
     make -j 8
     ldd libtaichi_core.so
     cd ../python
-    ti test -t 1
+    ti test -t 1 -na opengl
```

Comment:
Why exclude? I thought we support auto detect now so this is not required?

Reply:
Just for safety - the Linux buildbot does not have GUI, and I would like to minimize the #attempts to make the latest version build on that environment.